Exchange/SelectCurrencyView/SelectCurrencyViewController.swift

```diff
@@ -113,47 +95,40 @@ extension SelectCurrencyViewController: UITableViewDelegate {
         _ tableView: UITableView,
         didSelectRowAt indexPath: IndexPath
     ) {
-        bindings.didSelectCell(indexPath)
+        let cellViewModel = cellViewModels.value[indexPath.row]
+        didSelectCurrency.accept(cellViewModel)
     }
 }
 //MARK: - Implement ViewType
 
 extension SelectCurrencyViewController: ViewType {
     typealias ViewModel = SelectCurrencyViewModel
     
+    var bindings: ViewModel.Bindings {
+        .init(
+            didSelectCurrency: didSelectCurrency.asSignal(),
+            searchCurrency: searchController.searchBar.rx.text.asDriver()
```

Comment:
Наверное, это все-таки `searchText`
```suggestion
            searchText: searchController.searchBar.rx.text.asDriver()
```